docs/operating-scylla/admin-tools/scylla-sstable.rst

```diff
@@ -99,40 +96,529 @@ Dropped columns can be provided to the tool in the form of insert statements int
 
 System tables
-*************
+^^^^^^^^^^^^^
 
 If the examined table is a system table -- it belongs to one of the system keyspaces (``system``, ``system_schema``, ``system_distributed`` or ``system_distributed_everywhere``) -- you can just tell the tool to use the known built-in definition of said table. This is possible with the ``--system-schema`` flag. Example:
 
 .. code-block:: console
 
     scylla sstable dump-data --system-schema system.local ./path/to/md-123456-big-Data.db
 
+.. _scylla-sstable-sstable-content:
+
+Sstable Content
+---------------
+
+.. _SStable: /architecture/sstable
+
+All operations target either one specific sstable component or all off them as a whole.
+For more information about the sstable components and the format itself, visit SStable_.
+
+On a conceptual level, the data in sstables is represented by objects called mutation fragments. We have the following kinds of fragments:
+
+* ``partition-start`` (1) - represents the start of a partition, contains the partition key and partition tombstone (if any);
+* ``static-row`` (0-1) - contains the static columns if the schema (and the partition) has any;
+* ``clustering-row`` (0-N) - contains the regular columns for a given clustering row, if there are no clustering columns a partition will have exactly one of these;
+* ``range-tombstone-change`` (0-N) - contains a (either start or end) bound of a range deletion;
+* ``partition-end`` (1) - represents the end of the partition;
+
+Numbers in parantheses represent the number of the fragment type in a partition.
+
+Data from the sstable is parsed into these fragments.
+This format allows us to represent a small part of a partition or an arbitrary number of partition, even the entire content of an sstable.
+The ``partition-start`` and ``partition-end`` fragments are always present, even if a single row is read from a partition.
+If the stream contains multiple partitions, these follow each other in the stream, the ``partition-start`` fragment of the next partition following the ``partition-end`` fragment of the previous one.
+The stream is strictly ordered:
+* Partitions are ordered according to their token (hashes);
+* Fragments in the partition are ordered according to their order presented in the listing above, ``clustering-row`` and ``range-tombstone-change`` fragments can be intermingled, see below.
+* Clustering fragments (``clustering-row`` and ``range-tombstone-change``) are ordered between themselves according to the clustering order defined by the schema.
+
 Supported Operations
-^^^^^^^^^^^^^^^^^^^^^^^
-The ``dump-*`` operations output JSON. For ``dump-data``, you can specify another output format.
+--------------------
+
+dump-data
+^^^^^^^^^
+
+Dump the content of the data component. This component contains the data-proper
+of the sstable. This might produce a huge amount of output. In general the
+human-readable output will be larger than the binary file.
```

Comment:
```suggestion
Dumps the content of the data component (the component that contains the data-proper
of the SStable). This operation might produce a huge amount of output. In general, the
human-readable output will be larger than the binary file.
```

Reply:
I don't know what the "data-proper" of the SStable" is, so I'll leave it as is.

Reply:
The sstable has severeal components, most of it contain metadata, indexes etc. It is the data component that actually stores the data this sstable contains.